scipy/special/_legacy.pxd

```diff
@@ -135,8 +135,26 @@ cdef inline double smirnov_unsafe(double n, double e) nogil:
     _legacy_cast_check("smirnov", n, 0)
     return smirnov(<int>n, e)
 
+cdef inline double smirnovc_unsafe(double n, double e) nogil:
```

Comment:
As we discussed before, we don't want to add more unsafe versions of functions.

Reply:
Yes, I tried not adding the additional entries to `_legacy.pxd` but immediately incurred errors when testing `stats`.  These seem related to the signature and disappeared when I added the entries into `_legacy.pxd`.  E.g.
```
$ python runtests.py -n -j4 -s stats
...
self = <scipy.stats._continuous_distns.ksone_gen object at 0x10cd616a0>, q = array([0.5]), n = array([1.])

    def _ppf(self, q, n):
>       return sc.smirnovci(n, q)
E       TypeError: ufunc 'smirnovci' not supported for the input types, and the inputs could not be safely coerced to any supported types according to the casting rule ''safe''

n          = array([1.])
q          = array([0.5])
self       = <scipy.stats._continuous_distns.ksone_gen object at 0x10cd616a0>

../../scipy/stats/_continuous_distns.py:53: TypeError
```